swal-forms.js

```diff
@@ -219,7 +219,8 @@
         return inputTag + labelTag
 
         function toHtmlOptions (optionsString, option) {
-          return optionsString + t("<option value='{value}'>{text}</option>", option)
+            option.selected = ((typeof option.selected !== 'undefined') && option.selected) ? ' selected' : ''
```

Comment:
"((undefined) ? 1 : 2) == 2" evaluates to true on Chrome, so I think we are good removing the undefined check.